sway/sway.5.txt

```diff
@@ -361,7 +361,8 @@ The default colors are:
 
 **output** <name> <background|bg> <file> <mode>::
 	Sets the wallpaper for the given output to the specified file, using the given
-	scaling mode (one of "stretch", "fill", "fit", "center", "tile").
+	scaling mode (one of "stretch", "fill", "fit", "center", "tile"). This image
+	must be in PNG format unless sway was built with gdk_pixbuf.
```

Comment:
Well, you also improved the message in this pull request, so I'm not really concerned. You could extend the error message to include the likely root cause if you like.